setup.py

```diff
@@ -19,11 +19,11 @@
     include_package_data=True,
 
     install_requires=[
-        'tornado>=3.2.2',
-        'pyOpenSSL>=16.2.0',
-        'msgpack>=0.5.1',
-        'xxhash>=1.0.1',
-        'lmdb>=0.92',
+        'tornado>=3.2.2,<5.0.0',
+        'pyOpenSSL>=16.2.0,<18.0.0',
+        'msgpack>=0.5.1,<0.6.0',
```

Comment:
I set the max range for msgpack to be its next minor release because I think they are probably going to be making major changes again before 1.0.0.

Reply:
The plan for msgpack is to make those changes in a an incremental fashion, so we could probably bump that cap up as we know they make releases which are compatible with us.

Reply:
Yep. I'd rather have the tighter range in setup.py because we will still see the updates in synapse-base-image and can come back here and bump the maximum when we know it works.